opentelemetry-api/tests/metrics/integration_test/test_cpu_time.py

```diff
@@ -0,0 +1,116 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# type: ignore
+
+import io
+from typing import Iterable
+from unittest import TestCase
+
+from opentelemetry.metrics import _DefaultMeter
+from opentelemetry.metrics.measurement import Measurement
+
+# FIXME Test that the instrument methods can be called concurrently safely.
+
+
+class ChildMeasurement(Measurement):
+    def __init__(self, value, attributes=None):
+        super().__init__(value, attributes=attributes)
+
+    def __eq__(self, o: Measurement) -> bool:
+        return self.value == o.value and self.attributes == o.attributes
+
+
+class TestCpuTimeIntegration(TestCase):
```

Comment:
This test is basically the example code I put in the docstring above. Also shows the function callback form working